components/text-field.php

```diff
@@ -1,8 +1,7 @@
-<?php function insert_text_field($placeholder, $id, $required, $type = "text", $args = ""): void
+<?php function insertTextField($placeholder, $id, $required, $type="text", $value = "")
 { ?>
     <label for="<?= $id ?>" class="text-field-label">
         <?= $placeholder ?> <br>
-        <input class="text-field" type="<?= $type ?>" name="<?= $id ?>" placeholder="<?= $placeholder ?>"
-               id="<?= $id ?>" <?php if ($required) echo "required"; ?> <?= $args ?>>
+        <input class="text-field" type="<?= $type ?>" name="<?= $id ?>" placeholder="<?= $placeholder ?>" id="<?= $id ?>" value="<?= htmlspecialchars($value) ?>" <?php if ($required) echo "required"; ?>>
```

Comment:
```suggestion
        <input class="text-field" type="<?= $type ?>" name="<?= $id ?>" placeholder="<?= $placeholder ?>"
                     id="<?= $id ?>" value="<?= htmlspecialchars($value) ?>"
                     <?php if ($required) echo "required"; ?> <?= $args ?>>
```

This line was pretty big :smile: